proxy/coco/src/spawn_abortable.rs

```diff
@@ -26,6 +26,7 @@ pub enum Error {
 /// Stop-gap until we can abort [`JoinHandle`]s directly:
 /// tokio-rs@cbb14a7bb9a13363e1abee8caff2bad1f996c263
 #[allow(clippy::missing_docs_in_private_items)]
+#[must_use]
```

Comment:
We could maybe add a hint why the value needs to be used with `#[must_use = "..."]`.